sssd_test_framework/utils/tools.py

```diff
@@ -599,27 +599,31 @@ def __init__(self, host: MultihostHost) -> None:
         """
         super().__init__(host)
 
-    def passwd(self, name: str | int) -> PasswdEntry | None:
+    def passwd(self, name: str | int, service="") -> PasswdEntry | None:
         """
         Call ``getent passwd $name``
 
         :param name: User name or id.
         :type name: str | int
+        :param service: Service used.
+        :type service: str | Defaults to None, optional
```

Comment:
move "defaults to None" to param, type is type

I'm don't know what editor do you use. vscode has autodocstring plugin that does this for you.

Reply:
I use pycharms.